src/py/flwr/client/app.py

```diff
@@ -268,8 +269,16 @@ def start_numpy_client(
     >>>     root_certificates=Path("/crts/root.pem").read_bytes(),
     >>> )
     """
-    # Start
-    _check_actionable_client(client, client_fn)
+    warnings.warn(
+        "flwr.client.start_numpy_client() is deprecated and will"
+        "be removed in a future version of Flower.",
+        DeprecationWarning,
```

Comment:
Shouldn't this be the following?
```suggestion
        warnings.DeprecationWarning,
```